src/custodian/src/custodian_server.erl

```diff
@@ -143,7 +143,49 @@ handle_db_event(_DbName, _Event, _St) ->
     {ok, nil}.
 
 check_shards() ->
-    [?CUSTODIAN_MONITOR:send_event(Item) || Item <- custodian:summary()].
+    [send_event(Item) || Item <- custodian:summary()].
+
+
+send_event({_, Count} = Item) ->
+    Description = describe(Item),
+    Name = check_name(Item),
+    Level = case Count of
+        0 ->
+            "--ok";
+        1 ->
+            couch_log:critical("~s", [Description]),
+            "--critical";
+        _ ->
+            couch_log:warning("~s", [Description]),
+            "--warning"
+    end,
+    ?CUSTODIAN_MONITOR:send_event(Name, Level, Description).
```

Comment:
think we should send `Count` itself, the transformation to `Level` is sensu-specific. 😬